docs/developer/meta/config-specs.md

```diff
@@ -38,8 +38,9 @@ Every file has 3 possible attributes:
 
 - `name` - This is the name of the file the Agent will look for (**REQUIRED**)
 - `example_name` - This is the name of the example file the Agent will ship. If none is provided, the
-  default will be `conf.yaml.example`. The exception is auto-discovery files, which are also named
-  `auto_conf.yaml`.
+  default will be `conf.yaml.example`. The exceptions are:
```

Comment:
```suggestion
  default will be `conf.yaml.example`. The exceptions are as follows:
```
Using complete sentences to introduce lists makes our docs easier to translate into other languages.